src/chart/animator/animation.cpp

```diff
@@ -133,7 +127,7 @@ void Animation::addKeyframe(const Gen::PlotPtr &next,
 	addKeyframe(begin,
 	    next,
 	    options,
-	    strategy == RegroupStrategy::drilldown);
+	    strategy != RegroupStrategy::fade);
```

Comment:
I suspect that this change should be done for the first addKeyframe in line 117

Reply:
OK I'll do that in the next PR